packages/payments/src/constants/settings.ts

```diff
@@ -0,0 +1,31 @@
+const isDev = (): boolean => {
+  return !!(
+    typeof window !== "undefined" &&
+    window.localStorage.getItem("IS_PAPER_DEV") === "true"
+  );
+};
+
+const isOldPaperDomain = (): boolean =>
+  typeof window !== "undefined" && window.location.origin.includes("paper.xyz");
+
+export const getPaperOriginUrl = (): string => {
+  if (isDev()) {
+    return (
+      window.localStorage.getItem("PAPER_DEV_URL") ?? "http://localhost:3000"
+    );
+  }
+
+  if (isOldPaperDomain()) {
+    return window.location.origin;
+  }
+
+  return "https://payments.thirdweb.com";
+};
+
+// eslint-disable-next-line better-tree-shaking/no-top-level-side-effects
+export const PAPER_APP_URL = getPaperOriginUrl();
```

Comment:
if this is safe to rename then let's rename it now?

Reply:
This export shouldn't be exposed publicly, just internally